sormas-backend/src/main/java/de/symeda/sormas/backend/audit/AuditLoggerEjb.java

```diff
@@ -362,6 +357,154 @@ public void logFailedUiLogin(String caller, String method, String pathInfo) {
 		accept(uiLoginFail);
 	}
 
+	@Override
+	public void logGetExternalLabMessagesSuccess(Date since, List<String> externalLabMessages, Date start, Date end, String authAlias) {
+		Coding type = new Coding("https://hl7.org/fhir/R4/valueset-audit-event-type.html", "110107", "Import");
+		String outcome = String.format("%d external lab messages since %s fetched", externalLabMessages.size(), since);
+		Reference what = new Reference("getExternalLabMessages");
```

Comment:
How about defining constants for the values `110107`, `Import` and `getExternalLabMessages`? They are used four times.
Generally, I wonder whether it would make sense to use constants in this class. There are many String values used multiple times...

Reply:
fixed